packages/fluentui/react-builder/src/components/NarrationComputer.tsx

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
nit:
I suggest to move this file to another directory, for example `packages/fluentui/react-builder/src/narration`. Currently it stays in `packages/fluentui/react-builder/src/components/` where I expect too see only React components.

Reply:
Okay, moved as suggested. The change is included in [this PR](https://github.com/microsoft/fluentui/pull/15345).